packages/databyss-ui/theming/mediaBreakpoints.js

```diff
@@ -0,0 +1,4 @@
+export const mobileBreakpoint = { maxWidth: 425 }
```

Comment:
We already have a `/theming/responsive` file that looks similar. Can we adjust and use that?